src/template/related-search.ts

```diff
@@ -1,13 +1,40 @@
-import { html } from 'lit-html'
+import { html, render } from 'lit-html'
+import { urlParameterMap } from '../utils/urlParameter'
+import { fetchData } from '../utils/fetchData'
+import { mainTemplate } from './main'
+import { noResultsTemplate } from './no-results'
 
 export function relatedResultsTemplate (contextualNavigation: { categories: any; }) {
+  const onRelatedSearchClick = (e: { preventDefault: () => void; target: { href: any; textContent: any } }) => {
```

Comment:
probably you could define a type and reuse it in this file:
```
type ClickEvent = { preventDefault: () => void; target: { href?: string; textContent?: string } };
```
or just use `e: InputEvent`